tests/unit/zhmcclient/test_notification.py

```diff
@@ -126,21 +146,38 @@ def mock_add_message(self, headers, message):
 
     def mock_start(self):
         """Start the STOMP message sender thread."""
-        assert self._state_connected
+        # assert self._state_connected
```

Comment:
clean code: what is the purpose of commenting out code here (and below)?
If this is a temporary thing, consider to add a TODO / FIXME explanation.

If this is meant to stay in, add a reason why it is commented but needs to stay in the file.

Reply:
Thanks for catching this. This was in fact a try-out that I forgot to resolve in one or the other way. I think the mock_start/end functions should not be dependent on the connection state, and have removed the commented-out assertions.